.gitlab-ci.yml

```diff
@@ -95,22 +96,150 @@ whois-integration-test:
     - mariadb:10.2
   tags:
     - whois
+  stage: integration-test
+  rules:
+    - if: '$CI_COMMIT_REF_NAME == "master"'
+      when: always
+    - when: manual
+      allow_failure: true
+  script:
+    - mvn -DskipTests install
+    - mvn $MAVEN_OPTS -Ddb.host=mariadb verify -Pintegration -pl whois-api -T 1C
+
+whois-client-integration-test:
+  <<: *it_env
```

Comment:
Minor: Names are quite large it does not fit in small modal box of gitlab  . Can we rename to whois-client-test or something similar as the stage is itself Integration stage